### ISSUE TITLE
missing manifest's version number when installing via hacs

### ISSUE THREAD

**Issue by unlucio** [albertogeniola/meross-homeassistant#242]
**Describe the bug**
After installing via hacs, the component doesn't load and homeassistant's log complains about the manifest:

```
The custom integration 'meross_cloud' does not have a valid version key (None) in the manifest file 
```

checking the manifest under `homeassistant/custom_components/meross_cloud`
```bash
# cat manifest.json
{
  "domain": "meross_cloud",
  "name": "Meross Cloud IoT",
  "documentation": "https://www.home-assistant.io/components/meross_cloud",
  "dependencies": ["persistent_notification"],
  "codeowners": ["@albertogeniola"],
  "requirements": ["meross-iot==0.4.1.0rc7"],
  "config_flow": true,
  "quality_scale": "platinum"
}
```
the `version` property is actually missing while it's present in the [repo's manifest.js](https://github.com/albertogeniola/meross-homeassistant/blob/master/custom_components/meross_cloud/manifest.json)

Manually adding the missing property silences the error line, yet the integration still doesn't show up in the add menu.
Logs look fine besides the traditional custom integration warning.

**Your environment**
HomeAssistant version: 2021.6.0
Hassio Version (if applicable): hassio-supervisor 2021.05.4, hassio-cli 2021.05.1
Hardware environment: raspberry pi 3B+
**Logs taken when the issue happened**
```
2021-06-03 18:20:46 ERROR (SyncWorker_1) [homeassistant.loader] The custom integration 'meross_cloud' does not have a valid version key (None) in the manifest file and was blocked from loading. See https://developers.home-assistant.io/blog/2021/01/29/custom-integration-changes#versions for more details

---

2021-06-03 18:40:10 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration meross_cloud which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
```

**Comment by Raul-7-7** [comment]
Duplicate: https://github.com/albertogeniola/meross-homeassistant/issues/218

**Comment by albertogeniola** [comment]
Closing as it should be fixed by 0267655fbd01534d1b19af8f947467ec0d61ce19

**Comment by swythan** [comment]
Hi @albertogeniola, thanks for jumping on this so fast, but I'm afraid it's not working for me.

I was trying to work out how the versioning works, and AFAICT HACS uses the git commits from the GitHub releases. 
https://hacs.xyz/docs/publish/start/#versions

If that's right then I guess you'll need to make a new GH release for HACS to pick up any changes.

**Comment by Raul-7-7** [comment]
@swythan left comment for @albertogeniola on the commit as well. HACS is not going to detect it unless the version is bumped as you have in your doc.

**Comment by swythan** [comment]
Fixed for me in the 1.0.7 release. Thanks @albertogeniola 🎉

**Comment by albertogeniola** [comment]
You're welcome. Sorry for the late fix.
